Gemfile.lock

```diff
@@ -47,7 +47,7 @@ GEM
       multi_json (~> 1.0)
     addressable (2.3.5)
     arel (3.0.3)
-    bcrypt (3.1.7)
+    bcrypt (3.1.10)
```

Comment:
Running the following will create the same `Gemfile.lock` in `169c610`.

```bash
gem uninstall --all --force -x
git checkout 1989c18      # Previous commit to adding will_paginate_mongoid gem
gem install bundler --version=1.10.5
bundle
git checkout 169c610      # will_paginate_mongoid gem is now in slices.gemspec
bundle
```

We're adding a _new_ gem rather then updating an existing gem so that must be triggering a partial rebuild of `Gemfile.lock`.

Deleting the `Gemfile.lock` running `bundle` produces a very different file, with lots of updated gems such as `rails 3.2.22`.

Uninstalling `bcrypt` v3.1.10, manually editing `Gemfile.lock` to `bcrypt (3.1.7)` and running `bundle` will keep the 3.1.7 version. But the next time a `gem` is changed it'll probably update again so that'll only be a temporary fix.

I can't see why `3.1.10` is installed. Shall we keep it in?